package.json

```diff
@@ -29,33 +29,34 @@
   },
   "homepage": "https://github.com/samvera-labs/nectar-iiif.git#readme",
   "devDependencies": {
-    "@iiif/presentation-3": "^1.0.4",
-    "@iiif/vault": "^0.9.17",
-    "@testing-library/jest-dom": "^5.16.4",
-    "@testing-library/react": "^13.1.1",
-    "@types/jest": "^27.4.1",
-    "@types/react": "^18.0.5",
-    "@types/react-dom": "^18.0.1",
+    "@iiif/presentation-3": "^1.1.3",
+    "@iiif/vault": "^0.9.19",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@types/jest": "^29.2.3",
+    "@types/react": "^18.0.26",
+    "@types/react-dom": "^18.0.9",
     "@types/sanitize-html": "^2.6.2",
     "chokidar": "^3.5.3",
-    "esbuild": "^0.14.36",
-    "esbuild-envfile-plugin": "^1.0.2",
-    "fs-extra": "^10.1.0",
-    "jest": "^27.5.1",
-    "live-server": "^1.1.0",
-    "markdown-it": "^13.0.0",
+    "esbuild": "^0.15.16",
+    "esbuild-envfile-plugin": "^1.0.5",
+    "fs-extra": "^11.1.0",
+    "jest": "^29.3.1",
+    "jest-environment-jsdom": "^29.3.1",
+    "live-server": "^1.2.2",
+    "markdown-it": "^13.0.1",
     "rimraf": "^3.0.2",
-    "shiki": "^0.10.1",
-    "ts-jest": "^27.1.4",
-    "ts-node": "^10.7.0",
-    "typescript": "^4.6.3"
+    "shiki": "^0.11.1",
+    "ts-jest": "^29.0.3",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.9.3"
   },
   "dependencies": {
-    "@stitches/react": "^1.2.7",
-    "hls.js": "^1.1.5",
-    "react": "^16.8  || ^17.0 || ^18.0",
-    "react-dom": "^16.8  || ^17.0 || ^18.0",
-    "sanitize-html": "^2.7.0"
+    "@stitches/react": "^1.2.8",
+    "hls.js": "^1.2.8",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```

Comment:
This is okay, right? We had been using the ORs `"react": "^16.8  || ^17.0 || ^18.0"` but I just want to make sure...

Reply:
Ah, good catch.  I'll switch back.